test/fixtures/fhqwhgads/node_modules/fhqwhgads/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "fhqwhgads",
+  "version": "3.7.2",
```

Comment:
It won't matter here (since Node.js doesn't parse the version field) but in the bluebird case the version under `node_modules` was a `2.x` version (in other words it bootstraps its build script with an earlier version of the module).

Reply:
I think I'll just remove the "version" number entirely. Test still fails/passes as expected without it.